scripts/run_all_tests.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-if [ -z "$(ls | grep CONTRIBUTING.md)" ]; then
-  echo "Please run the script from repo directory"
-  exit -1
-else
-  echo "Running the tests"
-  python -m pytest --cov=gradio --durations=20 --durations-min=1 test
-fi
+
+cd "$(dirname ${0})/.."
+# source scripts/create_test_requirements.sh
```

Comment:
I don't think we want to create the test requirements each time you run the tests. It unnecessarily slows the process